packages/components/src/components/tabs/tabs/tabs.tsx

```diff
@@ -113,23 +106,19 @@ export class Tabs {
           gradientColor={this.gradientColor}
           activeTabIndex={this.activeTabIndex}
           onUpdate={this.onTabsBarUpdate}
-          onTabChange={(e) => e.stopPropagation()}
+          onTabChange={(e) => e.stopPropagation()} // prevent double event emission because of identical name
         >
           {this.tabsItemElements.map((tab, index) => (
             <button key={index} type="button">
               {tab.label}
             </button>
           ))}
         </PrefixedTagNames.pTabsBar>
-        <slot onSlotchange={this.onSlotChange} />
+        <slot onSlotchange={this.defineTabsItemElements} />
```

Comment:
Can we use `onSlotchange` here without having the same issue like with Tabs Bar?